configs/default/HARC-HAKRCF722V2.config

```diff
@@ -115,7 +115,8 @@ feature OSD
 serial 5 64 115200 57600 0 115200
 
 # master
-set gyro_to_use = BOTH
+# this does not work in 4.4.1
+# set gyro_to_use = BOTH
```

Comment:
This will break the target for 4.4.0

Reply:
Will it? It'll only disable 1 gyro.

Reply:
Cannot confirm as I don't have any dual gyro board for testing.